File: Other Algorithms/isPrime/isPrime.go
package main

import (
	"fmt"
)

func main() {
	status := isPrime(42)
	fmt.Println(status)
}

func isPrime(value int) bool {
	for i := 2; i <= value; i++ {
		if value%i == 0 && i != value {
			return false
		}
	}
	return true
}

File: Other Algorithms/isPrime/isPrime.py
def is_prime(x):
    if x < 2:
        return False
    if x > 2 and x % 2 == 0:
        return False
    for i in range(3, int(x ** 0.5), 2):
        if x % i == 0:
            return False
    return True


def main():
    x = int(input())
    print(str(x) + " is prime: " + str(is_prime(x)))


if __name__ == '__main__':
    main()
